eng/Version.Details.xml

```diff
@@ -2,33 +2,33 @@
 <Dependencies>
   <ProductDependencies></ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.18519.9">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.18522.6">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>79bb729e1447a75e5b59bf7184e324db41b3f02e</Sha>
+      <Sha>dd701110d79bb60c84a141ab9017320ed2df0b41</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.ApiCompat" Version="1.0.0-beta.18519.9">
+    <Dependency Name="Microsoft.DotNet.ApiCompat" Version="1.0.0-beta.18522.6">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>79bb729e1447a75e5b59bf7184e324db41b3f02e</Sha>
+      <Sha>dd701110d79bb60c84a141ab9017320ed2df0b41</Sha>
     </Dependency>
     <Dependency Name="Microsoft.DotNet.GenApi" Version="1.0.0-beta.18478.5">
```

Comment:
@alexperovich @mmitche Any ideas on why this isn't updated? It listed in the packages updated as `Microsoft.DotNet.GenAPI - 1.0.0-beta.18522.61`. Is this a string casing issue? Should we ignore case for package names and simply normalize them with the update?

Reply:
@weshaggard Names should be case insensitive now, and normalized by the update, at least in source.  Not sure whether those fixes have rolled out yet though.